docs/index.asciidoc

```diff
@@ -31,6 +31,20 @@ Files ending in `.gz` are handled as gzip'ed files.
 
 Files that are archived to AWS Glacier will be skipped.
 
+==== Event Metadata and the Elastic Common Schema (ECS)
+This plugin adds cloudfront metadata to event.
+By default, the value is stored in the root level.
+When ECS is enabled, it is stored in the `@metadata`.
+
+Here’s how ECS compatibility mode affects output.
+[cols="<l,<l,e,<e"]
+|=======================================================================
+| disabled | v1 | Availability | Description
+
+| cloudfront_fields | [@metadata][s3][cloudfront][fields] | available when the file is a cloudfront metadata | columns name of metadata
```

Comment:
```suggestion
| cloudfront_fields | [@metadata][s3][cloudfront][fields] | available when the file contains cloudfront metadata | columns name of metadata
```

Reply:
I don't understand this wording.  Maybe "When the file _contains_ cloudfront metadata?"  
If I don't have the wording right, please let me know. We can work together to find the right wording.

Reply:
my word `metadata` is not accurate. It should be a log file specific to CloudFront, which records the access details.
`cloudfront_fields` and `cloudfront_version` are only available for CloudFront log.
So, may be change it to "available when the file is a CloudFront log"